dags/bqetl_subplat_backfill.py

```diff
@@ -0,0 +1,51 @@
+import datetime
+
+from airflow import DAG
+
+from operators.gcp_container_operator import GKEPodOperator
+
+docs = """
+### bqetl_subplat_backfill
+
+Temporary DAG for backfilling Python ETLs which are normally run in [`bqetl_subplat`](https://github.com/mozilla/bigquery-etl/blob/generated-sql/dags/bqetl_subplat.py).
+
+#### Owner
+
+srose@mozilla.com
+"""
+
+default_args = {
+    "owner": "srose@mozilla.com",
+    "email": ["telemetry-alerts@mozilla.com", "srose@mozilla.com"],
+    "depends_on_past": False,
+    "retry_delay": datetime.timedelta(seconds=120),
+    "email_on_failure": True,
+    "email_on_retry": False,
+    "retries": 2,
+}
+
+tags = ["impact/tier_1"]
+
+with DAG(
+    "bqetl_subplat_backfill",
+    default_args=default_args,
+    schedule_interval="@daily",
```

Comment:
do we need the schedule interval here? sounds like this gets kicked of manually once for the backfill

Reply:
Yes, because I want it to automatically trigger a DAG run for every day between the start and end dates, and the interval is how it determines what DAG runs to automatically schedule.  Without the schedule interval I'd have to manually trigger the DAG for each day.